examples/planning/get_ad_group_criterion_cpc_bid_simulations.rb

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env ruby
+
+#
+# Copyright:: Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example gets all available ad group criterion CPC bid simulations for a
+# given ad group. To get ad groups, run get_ad_groups.rb.
+
+require 'optparse'
+require 'google/ads/google_ads'
+
+def get_ad_group_criterion_cpc_bid_simulations(customer_id, ad_group_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  query = <<~QUERY
+    SELECT ad_group_criterion_simulation.ad_group_id,
+           ad_group_criterion_simulation.criterion_id,
+           ad_group_criterion_simulation.start_date,
+           ad_group_criterion_simulation.end_date,
+           ad_group_criterion_simulation.cpc_bid_point_list.points
+    FROM ad_group_criterion_simulation
+    WHERE ad_group_criterion_simulation.type = CPC_BID 
+    AND ad_group_criterion_simulation.ad_group_id = #{ad_group_id}
+  QUERY
+
+  responses = client.service.google_ads.search_stream(
+    customer_id: customer_id,
+    query: query
```

Comment:
nit: we usually put a comma at the end of last parameter too, so that future changes will have the minimal lines of diff

Reply:
Sounds good, thanks.